### ISSUE TITLE
git lfs threshold not applied/working

### ISSUE THREAD

**Issue by benj919** [SwissDataScienceCenter/renku#1618]
**Describe the bug**
According to the docs it's possible to set a file size limit above which any file is automatically stored with git lfs.
https://renku.readthedocs.io/en/latest/user/data.html
this does not seem to work. (Also there is a typo? with the renku config interface. I think the "set" is no longer needed.) 
We have a series of lecture projects, and from time to time someone manages to push large dataset files into the repos. 

**Link to project**
Any (new) project

**To Reproduce**
 ```
# Create a new project and environment

# set the file limit to e.g. 5mb: 
renku config lfs_threshold 5mb

# confirm the limit:
renku config lfs_threshold
# 5mb

# push and restart environment for the threshold to take effect (unsure if needed)
git push

# create/upload a file larger than that:
dd if=/dev/urandom of=testfile bs=1024 count=10240

# add, commit, and push the file:
git add testfile && git commit -m 'added testfile' && git push

# check if file is in git lfs, (I have used a new environment built from the file commit to check here)
git lfs ls-files
# nothing here

```
**Expected behavior**
Either the file being put into git lfs automatically according to the documentation, or the git commit/push being refused.

**Screenshots and/or execution output**
the above code can be run in the terminal, there is no unexpected or helpful output.

**Run environment (please complete the following information):**
- On renkulab.io
- jupyterlab
- Docker image base: renku/renkulab-r:4.0.0-renku0.10.4-0.6.3 (current R image on project creation)
- firefox

**Comment by ableuler** [comment]
Hi @benj919,
and thanks for reporting this issue. The problem here is that in order to check for these size limits, the githooks need to be actvated on the cloned repository. This can be done manually by executing `renku githooks install` as the first thing after launching an interactive environment on renkulab.io. 

We do this per default in our more [recent images](https://github.com/SwissDataScienceCenter/renkulab-docker/releases/tag/0.7.0) in the entrypoint of the container. So if you don't want to execute the `renku githooks install` command manually, you can use the newer version`renku/renkulab-r:4.0.0-0.7.0` as a base image in your Dockerfile. Please note that this base image comes with a newer version of the renku command line interface which, at this point, has a few incompatibilities with component versions that are currently deployed on renkulab.io. However, this will change very soon as we're in the process of rolling out a new version of RenkuLab to renkulab.io. 

So in short my recommendation is: 
- For the next few days, manually activate the githooks through `renku githooks install`.
- A week from now you will be able to safely switch to using `renku/renkulab-r:4.0.0-0.7.0` as base image.
